src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -171,6 +171,7 @@ their names (or descriptions) depend on some threshold.  Those filters are not i
     // Symbolic alleles
     public final static String SYMBOLIC_ALLELE_DEFINITION_HEADER_TAG = "ALT";
     public final static String NON_REF_SYMBOLIC_ALLELE_NAME = "NON_REF";
+    public final static Allele NON_REF_SYMBOLIC_ALLELE = Allele.create("<"+NON_REF_SYMBOLIC_ALLELE_NAME+">", false); // represents any possible non-ref allele at this site
```

Comment:
This exists already `Allele.NON_REF_ALLELE`

Reply:
Done.